src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -136,7 +136,7 @@ public bool EmbeddedPortablePdbHasLocalSource(string module, out string firstNot
         public bool PortablePdbHasLocalSource(string module, out string firstNotFoundDocument)
         {
             firstNotFoundDocument = "";
-            using (var moduleStream = File.OpenRead(module))
+            using (var moduleStream = _fileSystem.OpenRead(module))
```

Comment:
missing code line 147 do a search for `FileStream` in this file

Reply:
Done